Source/ThirdParty/GltfPipeline/parseGlb.js

```diff
@@ -72,6 +72,7 @@ import RuntimeError from '../../Core/RuntimeError.js'
             if (defined(binaryGltfBuffer)) {
                 binaryGltfBuffer.extras._pipeline.source = binaryBuffer;
             }
+            delete binaryGltfBuffer.uri;
```

Comment:
Will open a separate PR into gltf-pipeline for this fix.

Reply:
https://github.com/CesiumGS/gltf-pipeline/pull/578

Reply:
@lilleyse does that PR hold up this one?

Reply:
That PR doesn't need to hold this one up, we can merge this and bring in the gltf-pipeline changes at a later point.